### PR TITLE
[DOCS-6966] Updates to ACS 7.3 Install section

### PR DESCRIPTION
**Pull request by Amy-Hyland**
Applies updates for the distribution zip installation method.

⚠️ Updates for the Enterprise Docker Compose (via the Download Trial), Helm and Ansible deployments will be addressed separately.

### CODE REVIEW

**Comment by Amy-Hyland** [comment]
@HeinRagas PR of this section is ready for Review

File: content-services/latest/install/containers/docker-compose.md [hunk 1]
```diff
@@ -397,7 +397,7 @@ The Docker Compose file provides some default configuration. This section lists
 | FILE_STORE_URL | Shared file store URL (in this case, the name of the container is used) |
 | TRANSFORM_ENGINE_REQUEST_QUEUE | Name of the queue. The default value is `org.alfresco.transform.engine.aio.acs` |
 | PDFRENDERER_EXE | Location of the PDF Renderer binary. The default value is `/usr/bin/alfresco-pdf-renderer` |
-| LIBREOFFICE_HOME | Location of the LibreOffice installation. The default value is `/opt/libreoffice7.2` |
```
Comment (HeinRagas) [inline content-services/latest/install/containers/docker-compose.md:400]: Are we supporting LibreOffice 7.3 now, or is this the result of a mass find/replace?

Comment (Amy-Hyland) [reply content-services/latest/install/containers/docker-compose.md:400]: That should still support LibreOffice 7.2

**Review by Amy-Hyland — COMMENTED** [review]
Changed Libreoffice back to 7.2

**Comment by HeinRagas** [comment]
Yep, seems good to me.

File: content-services/latest/install/containers/docker-compose.md [hunk 2]
```diff
@@ -110,11 +110,11 @@ Use this information to verify that the system started correctly, and to clean u
...
         acs-trial_digital-workspace-1    quay.io/alfresco/alfresco-digital-workspace    2.6.0               1a2eaa5bf7a9        572.9 MB
         acs-trial_postgres-1             postgres                                       13.3                b2fcd079c1d4        314.7 MB
         acs-trial_proxy-1                alfresco/alfresco-acs-nginx                    3.2.0               da6d34dd9386        21.86 MB
-        acs-trial_share-1                quay.io/alfresco/alfresco-share                7.2.0               837b363e02af        758.7 MB
+        acs-trial_share-1                quay.io/alfresco/alfresco-share                7.3.0               837b363e02af        758.7 MB
         acs-trial_shared-file-store-1    quay.io/alfresco/alfresco-shared-file-store    0.16.1              dee75e9ffa5b        651.2 MB
         acs-trial_solr6-1                quay.io/alfresco/search-services               2.1.0               5800e8a31bdd        890.8 MB
         acs-trial_sync-service-1         quay.io/alfresco/service-sync                  3.5.0               7c0cee15f516        703.2 MB
```
Comment (anxumalo) [inline content-services/latest/install/containers/docker-compose.md:120]: TODO: verify versions in final release

**Review by anxumalo — APPROVED** [review]

**Pull request merged**